test/dlc_tests/container_tests/bin/pytorch_tests/testTorchdata

```diff
@@ -19,7 +19,7 @@ git clone $TORCHDATA_REPO \
 sed -i "s/awscli.*//g" data/test/requirements.txt
 
 echo "Installing pre-requisites."
-pip install -r data/test/requirements.txt
+pip install -r data/test/requirements.txt "awscli>=1.29.8" pyyaml==6
```

Comment:
awscli and pyyaml should be present on the container, why are we installing it explicitly?

Reply:
rolling it back as i see test passes without my changes now